features/taxation/applying_taxes_based_on_order_item_units_calculation_strategy/applying_correct_taxes_for_item_units_with_discount_for_item_when_tax_rate_included_in_price.feature

```diff
@@ -0,0 +1,50 @@
+@applying_taxes
+Feature: Applying correct taxes for item units with a discount when tax rates are included in price
+    In order to pay proper amount when buying goods
+    As a Visitor
+    I want to have correct taxes applied when the order is discounted and tax rates are included in products prices
```

Comment:
As above 👆🏼

Reply:
I'm resolving these comments, as they litter the PR a bit, the main comment is enough